airflow/providers_manager.py

```diff
@@ -571,10 +571,10 @@ def _add_taskflow_decorator(self, name, decorator_class_name: str, provider_pack
         if name in self._taskflow_decorators:
             try:
                 existing = self._taskflow_decorators[name]
-                other_name = f'{existing.__module__}.{existing.__name}'
+                other_name = f'{existing.__module__}.{existing.__name__}'
             except Exception:
                 # If problem importing, then get the value from the functools.partial
-                other_name = self._taskflow_decorators._raw_dict[name].args[0]
+                other_name = self._taskflow_decorators._raw_dict[name].args[0]  # type: ignore
```

Comment:
Not sure which error is silenced here but we started to silence specific error codes recently: https://mypy.readthedocs.io/en/stable/error_code_list.html#error-code-list

Reply:
Ahhhhh YEAH!